app/components/round-resource.js

```diff
@@ -16,5 +16,9 @@ export default Component.extend({
 
   score: computed('users.user.id', 'scorecards.user.id', function() {
     return this.get('scorecards').findBy('user.id', 4);
-  })
+  }),
+
+  click() {
```

Comment:
`link-to` seems pretty sufficient, right? Do we get any benefits from using `click()`?